tests/integration/test_remoted/test_multi_groups/test_merged_mg_file_content.py

```diff
@@ -0,0 +1,202 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Whenever a file is created or deleted from the group directory the merged.mg file must be updated. This test
+       checks the content of the merged.mg file that wazuh-remoted compiles for multi-groups.
+
+tier: 0
+
+modules:
+    - remoted
+
+components:
+    - manager
+
+daemons:
+    - wazuh-remoted
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-remoted.html
+
+tags:
+    - remoted
+'''
+import hashlib
+import os
+import re
+import subprocess as sb
+from time import sleep
+
+import pytest
+import requests
+from wazuh_testing.api import get_api_details_dict
+from wazuh_testing.remote import DEFAULT_TESTING_GROUP_NAME, new_agent_group, \
+                                  remove_agent_group
+from wazuh_testing.tools import REMOTE_DAEMON, WAZUH_PATH, configuration
+from wazuh_testing.tools.file import delete_file
+import wazuh_testing.tools as tools
+from wazuh_testing.tools.services import check_daemon_status, control_service
+from wazuh_testing.tools.wazuh_manager import remove_agents
+
+# Marks
+pytestmarks = [pytest.mark.linux, pytest.mark.server, pytest.mark.tier(level=0)]
```

Comment:
Only the server mark is needed here, the `pytest.mark.linux` mark is left over because the server can only be **linux**. And the tier must be specified by each test (above the test definition, see the comment below.).

```suggestion
pytestmarks = [pytest.mark.server]
```